.travis.yml

```diff
@@ -1,33 +1,34 @@
+language: python
+
+python:
+  - "2.7"
+
 addons:
   apt:
     packages:
       - expect-dev  # provides unbuffer utility
-      - python-lxml # because pip installation is slow
+      - python-lxml  # because pip installation is slow
 
-language: python
-python:
-  - "2.7"
+env:
+  global:
+  - VERSION="7.0" TESTS="0" LINT_CHECK="0"
```

Comment:
@eLBati @yvaucher that row seems not ivalidate the travis build, see https://travis-ci.org/OCA/l10n-italy/jobs/63628042 as reference.